src/products/firefox_desktop/fluent_migrations.md

```diff
@@ -0,0 +1,119 @@
+* Add new Fluent strings or files.
+* Remove obsolete strings.
+* Include a migration recipe.
+
+Once such patch lands in `mozilla-central`, it will be converted as part of the cross-channel generation, and pushed to a quarantine repository (`gecko-strings-quarantine`). At this point, this new content is not visible yet in [Pontoon](https://pontoon.mozilla.org/).
+
+The goal of migrations is to port existing translations to the new format without forcing community to retranslate them. For that to happen, migrations need to be performed before the new strings are pushed to the official `gecko-strings` repository, and become available in Pontoon.
+
+This means that patches including migrations should only be pushed from the quarantine repository to `gecko-strings` following the steps described in this document.
+
+## Running Fluent migrations
+
+This section of the document describes in detail which operations need to be performed in order to run migrations on over 150 Mercurial repositories for Firefox.
+
+### Prerequisites
+
+Several access rights are required:
+* Write access to [l10n-central](https://hg.mozilla.org/l10n-central), i.e. SSH access with `active_scm_l10n` rights. It’s possible to check your *Access Groups* in [your profile on people.mozilla.org](https://people.mozilla.org/e#nav-access-groups).
+* Sign-off rights in [Elmo](https://l10n.mozilla.org/).
+* Admin access to [Pontoon](https://pontoon.mozilla.org/).
+* An account on Heroku with access to [Pontoon apps](https://dashboard.heroku.com/apps/mozilla-pontoon/resources).
+
+In order to run migrations:
+* Install the [fluent.migrate](https://pypi.org/project/fluent.migrate/) Python package.
+* Clone all [l10n-central](https://hg.mozilla.org/l10n-central) repositories.
+* Clone [gecko-strings-quarantine](https://hg.mozilla.org/users/axel_mozilla.com/gecko-strings-quarantine/) and [gecko-strings](https://hg.mozilla.org/l10n/gecko-strings).
+
+A set of scripts to automate cloning and updating of the l10n-central repositories is [available here](https://github.com/flodolo/scripts/tree/master/mozilla_l10n/clone_hgmo).
+
+### Ensure there are no blocking issues in gecko-strings-quarantine
+
+As part of migrations, content needs to be pushed from the quarantine repository to `gecko-strings`. Since content can’t be pushed selectively, any pending issue needs to be solved before starting the process. For more information about the review process, see [this document](../review/review.md).
+
+### Clean up existing sign-offs
+
+Migrations will create new changesets in each of the repositories, resulting in new sign-offs to review for the PM in charge of Firefox or l10n-drivers.
+
+Before starting, it’s highly recommended to review all pending sign-offs in Elmo for the current Firefox Beta. More details about the sign-off process are available in [this document](../review/signoffs.md).
```

Comment:
The only alternative I can think of is "review pending sign-offs" -> "sign off available changesets". In this part:

> Before starting, sign off in Elmo all available changesets for the current Firefox Beta in the *Bulk Sign-offs* view.

And link to https://mozilla-l10n.github.io/documentation/products/review/signoffs.html#bulk-sign-offs in the last link.

In the next paragraph, remove *pending*.

Reply:
Yeah, that would work.

Reply:
Updating this and the sign-off doc to remove mention of *pending*.

I'm keeping "Firefox Beta" here for sake of clarity, but I plan to update the sign-off document in a follow-up, to explain that you can sign off central while Beta is string frozen.